.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@ default_language_version:
   python: python3
 repos:
 - repo: https://github.com/psf/black
-  rev: 24.10.0
+  rev: 24.8.0
```

Comment:
curious question 🤔 we are downgrading the black version?

Reply:
Yes! I responded to Jay's comment above for the context / explanation! 
But the tldr, the jenkins test was throwing up errors with the new version 24.10.0 since the version of python that kept on appearing when I was running it was python3.8 for some reason and 24.10.0 deprecates the usage of python3.8

Reply:
yep we need to upgrade the python version for Turnpike ... one day